sys/usb/usbus/cdc/ecm/cdc_ecm_netdev.c

```diff
@@ -142,7 +142,7 @@ static int _recv(netdev_t *netdev, void *buf, size_t max_len, void *info)
         return pktlen;
     }
     if (pktlen <= max_len) {
-        memcpy(buf, cdcecm->in_buf, pktlen);
+        memcpy(buf, cdcecm->data_out, pktlen);
```

Comment:
in_buf -> data_out ?

Reply:
Yeah, this was confusing before. I try to stick to USB terminology here. So OUT is from host to device and IN is from device to host. In this case the `data_out` is the buffer to receive data from the host, which is then copied to the receive buffer of netif.